Config.h.in

```diff
@@ -0,0 +1,2 @@
+#define passgen_VERSION_MAJOR 1
```

Comment:
Replace Hardcoded values here with the variables defined in the top-level CMakeLists.txt as

```
#define passgen_VERSION_MAJOR @passgen_VERSION_MAJOR@
#define passgen_VERSION_MINOR @passgen_VERSION_MINOR@
#define passgen_VERSION_REVISION @passgen_VERSION_REVISION@
```